modules/installation-special-config-encrypt-disk-tang.adoc

```diff
@@ -17,7 +17,7 @@ for a presentation on Tang.
 . Add kernel arguments to configure networking when you do the RHCOS installations for your cluster.
 If you miss this step, the second boot will fail.
 For example, to configure DHCP networking, identify `ip=dhcp`
-or set static networking when you add parameters to the kernel command line.
+or set static networking when you add parameters to the kernel command line. See the kernelArguments shown in the MachineConfig examples below.
```

Comment:
Also, the newly added sentence above I think suggests that there are some example kernel parameters set under `kernelArguments` that either configure DHCP (`ip=dhcp`) or static networking.

The only kernel argument that is included in the YAML example in this module is `rd.neednet=1`. My understanding is that this kernel argument enables networking even when `netroot` (utilities for network root file systems) is not available.  I'm assuming that in the context of Tang that is required so that the asymmetric keys can be obtained from the Tang server early on in the boot process. But, I could be wrong because I am new to Tang. @jlebon or @cgwalters please can you clarify this?

Essentially, I think we should be more explicit in the above sentence about what examples we are providing and what they do.

Reply:
Would it be useful to add a DHCP or static network configuration example too, since we are referring to that?

Reply:
Essentially, `rd.neednet=1` is required and then anything else on top of that that might be necessary if it's not just vanilla DHCP. So maybe we can reword this whole section as e.g.

> . Add kernel arguments to configure networking in the initramfs when you do the RHCOS installations for your cluster.
If you miss this step, the second boot will fail. To do this, you must add `rd.neednet=1` to the `kernelArguments` below. Additionally, you may need to add other arguments depending on your network setup. DHCP networking should work fine, but for example static networking would require specifying it via an `ip=` arg as documented in `dracut.cmdline(7)`.